wallet/model/model.go

```diff
@@ -5,14 +5,15 @@ import "time"
 type Profile struct {
 	Id       string `json:"id"`
 	Username string `json:"username,omitempty"`
-	AvatarId string `json:"avatar_id"`
+	AvatarId string `json:"avatar_id,omitempty"`
 }
 
 const ThumbnailWidth = 300
 
 type Metadata struct {
 	Version  string    `json:"version"`
-	Username string    `json:"username,omitempty"`
+	PeerId   string    `json:"peer_id"`
```

Comment:
Adding peer id here because it's needed in order to resolve a peer's profile. I think we need to rethink the metadata structure here, ideally it would be a traversable DAG, but then we'd have to encrypt each link separately, and the link names would be plaintext... trade offs.